.github/workflows/ci.yaml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
```

Comment:
IDE formatted the file — seems reasonable to be consistent in here anyway.

Reply:
Can you format with Prettier? I think it will be the same.